Source/WTF/wtf/PlatformEnable.h

```diff
@@ -1006,3 +1006,12 @@
     || PLATFORM(VISION)
 #define ENABLE_ACCESSIBILITY_ANIMATION_CONTROL 1
 #endif
+
+#if !defined(ENABLE_ADVANCED_PRIVACY_PROTECTIONS) \
+    && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
```

Comment:
Would this _disable_ advanced privacy protections on downlevels?

Reply:
I might be misunderstanding how this flag is supposed to work. HAVE_SYSTEM_SUPPORT_FOR_ADVANCED_PRIVACY_PROTECTIONS sounds obviously macOS 14+ only, but is this one supposed to match _any_ version of macOS we support? If so, maybe this can just be a `PLATFORM(MAC)`.